azavea/tilegarden
# Add `the_geom_webmercator` column to the example data

## Overview

As noted on PRs #12 and #13, Windshaft needs a `the_geom_webmercator` column in the source database.  This adds that column to the SQL dump used to populate the example database.

I made it by spinning up the database service and running the commands noted on the PRs:
```sql
ALTER TABLE pa_gardens ADD COLUMN the_geom_webmercator geometry(POINT, 3857);
UPDATE pa_gardens SET the_geom_webmercator = ST_Transform(ST_SetSRID(geom, 4326), 3857);
```
Then I exported the resulting table and gzipped it:
```
docker-compose exec -u postgres db pg_dump -d postgres -t pa_gardens > data/pa_gardens.sql
gzip data/pa_gardens.sql
```

## Testing Instructions

- Destroy your existing containers: `docker-compose down`
- Build and initialize them again: `./scripts/setup`
- Run them: `./scripts/server`
The demo should work, as described in PR #13, and you can use `./scripts/console db` to get a psql shell and examine the new column directly.
